contrib/forms/clinical_notes/new.php

```diff
@@ -137,7 +130,7 @@ function cbcell($name, $desc, $colname)
 
 <body <?php echo $top_bg_line;?> topmargin="0" rightmargin="0" leftmargin="2"
  bottommargin="0" marginwidth="2" marginheight="0">
-<form method="post" action="<?php echo $rootdir ?>/forms/clinical_notes/new.php?id=<?php echo $formid ?>"
+<form method="post" action="<?php echo attr($rootdir) ?>/forms/clinical_notes/new.php?id=<?php echo attr($formid) ?>"
```

Comment:
Don't html escape full paths (sometimes weird things can happen). Leave $rootdir alone, but the attr() around $formid is correct.